xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/RenamePageIT.java

```diff
@@ -366,18 +366,15 @@ void renamePageUsedInMacroContentAndParameters(TestUtils setup, TestReference te
 
         setup.rest().delete(testReference);
 
-        ViewPage standardLinkPage = setup.createPage(sourcePageReference1, "Some content to be linked. number 1");
-        ViewPage standardMacroLinkPage =
-            setup.createPage(sourcePageReference2, "Some content to be linked in macro. number 2");
-        ViewPage nestedMacroLinkPage =
-            setup.createPage(sourcePageReference3, "Some content to be linked in nested macro. number 3");
+        setup.createPage(sourcePageReference1, "Some content to be linked. number 1");
```

Comment:
Better sure the rest API if you don't need a ViewPage, it's much faster and reliable.